mempool/src/feerate.rs

```diff
@@ -0,0 +1,54 @@
+use common::primitives::amount::Amount;
+
+lazy_static::lazy_static! {
+    pub(crate) static ref INCREMENTAL_RELAY_FEE_RATE: FeeRate = FeeRate::new(1000);
+    pub(crate) static ref INCREMENTAL_RELAY_THRESHOLD: FeeRate = FeeRate::new(500);
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct FeeRate {
+    tokens_per_kb: u128,
+}
+
+impl FeeRate {
+    pub(crate) fn new(tokens_per_kb: u128) -> Self {
+        Self { tokens_per_kb }
+    }
```

Comment:
It would be nice if the name explicitly stated the unit to avoid confusion, maybe something like `from_tokens_per_kb`.

Reply:
I would also prefer using Amount as parameter

Reply:
Done